packages/flutter_tools/lib/src/base/io.dart

```diff
@@ -41,7 +41,6 @@ export 'dart:io'
         HttpServer,
         HttpStatus,
         pid,
-        Platform,
```

Comment:
you should probably add a comment to this file that talks about how you should avoid adding Platform, File, etc, to the export list, and to instead use... etc.